origins_workflow/origins_workflow.module

```diff
@@ -160,7 +160,11 @@ function origins_workflow_views_pre_render(ViewExecutable $view) {
       // page for un-privileged users.
       $account_uid = \Drupal::currentUser()->id();
       $account = User::load($account_uid);
-      if (!$account->hasPermission('use editorial transition publish')) {
+      // May reinstate this line to check 'use editorial transition publish'
+      // permission once bulk operation bug fixed - see D8NID-371 Jira ticket.
+      //
+      // if (!$account->hasPermission('use editorial transition publish')) {
```

Comment:
I think you can shorten this:

```
$account_uid = \Drupal::currentUser()->id();
$account = User::load($account_uid);
$account->hasPermission(....);
```

to this:

```
if (\Drupal::currentUser()->hasPermission(....)) { .... }
```

Reply:
Caveat: haven't looked the docs, so could be wrong! I'd be surprised if you need a full-fat User object to check this, though

Reply:
Hmm, interesting.

You are quite correct about \Drupal::currentUser()->hasPermission() which is a neat trick (adding it to my notes right now).
You will notice, though, that the line you are referring to is commented out (but have amended the code anyway in case it is ever reinstated).

It wasn't quite so easy with the role check, as it isn't possible to do \Drupal::currentUser()->hasRole() in the same way. 
It is possible, however, to get a list of the roles using \Drupal::currentUser()->getRoles() , so have amended the code accordingly.

Reply:
Scratch that, couldn't get the commented line of code to pass the tests so have removed it and put a wordy comment instead !